src/net_processing.cpp

```diff
@@ -491,47 +491,47 @@ struct CNodeState {
     //! The peer's address
     const CService address;
```

Comment:
This `address` member is now redundant and can be removed (from here and the initializer list). I'll do that in a follow-up.

Reply:
Fixed in #21395